.slicemachine/assets/customtypes/home/mocks.json

```diff
@@ -1,111 +1,302 @@
 {
-  "id": "mock-doc-id",
-  "url": null,
-  "type": "home",
-  "href": null,
-  "tags": [],
-  "first_publication_date": "1970-01-01T00:00:01+0000",
-  "last_publication_date": "1970-01-01T00:00:01+0000",
-  "slugs": [],
-  "linked_documents": [],
-  "lang": "en-us",
-  "alternate_languages": [],
-  "data": {
-    "title": "did",
-    "description": "said",
-    "hero_title": "tea",
-    "hero_description": "own",
-    "hero_actions": [
+  "uid": {
```

Comment:
Can you please provide some more context for this change? It's not clear how it was generated, and the content doesn't seem related to Open Visualization.